### DISCUSSION TITLE
Keenetic Giga KN-1011 need help

### DISCUSSION THREAD

**Discussion by dsm150** [Ysurac/openmptcprouter#2767] [Q&A]
Maybe someone has already tried to compile the firmware for the device? I will be very grateful for help. Since in this direction I am a complete 0 (((

**Comment by heini66** [comment]
First off all if you want to use OMR, check out if there is an Openwrt support for the device.
https://openwrt.org/toh/hwdata/keenetic/
There is no Openwrt support for this device.
https://downloads.openwrt.org/snapshots/targets/ramips/mt7621/
So you need to create device support for the KN-10011. Once this is done you might have a chance to get OMR for the KN-1011 compiled.
